staging/src/k8s.io/pod-security-admission/test/fixtures_procMount.go

```diff
@@ -28,7 +27,7 @@ func init() {
 		expectErrorSubstring: "procMount",
 		generatePass: func(p *v1.Pod) []*v1.Pod {
 			p = ensureSecurityContext(p)
-			return []*corev1.Pod{
+			return []*v1.Pod{
```

Comment:
We almost always prefer "somethingv1" to "v1" - so "corev1" is the better choice.  I saw a couple others like this.